packages/react-native-web/src/modules/createDOMProps/index.js

```diff
@@ -187,8 +187,16 @@ const createDOMProps = (component, props, styleResolver) => {
   // https://mathiasbynens.github.io/rel-noopener/
   // Note: using "noreferrer" doesn't impact referrer tracking for https
   // transfers (i.e., from https to https).
-  if (component === 'a' && domProps.target === '_blank') {
-    domProps.rel = `${domProps.rel || ''} noopener noreferrer`;
+  // Note: Specifying rel="opener" will explicitly bypass this safeguard.
+  if (component === 'a' && domProps.target === '_blank' && domProps.rel !== 'opener') {
+    const relList = domProps.rel ? domProps.rel.split(' ') : [];
+    if (relList.indexOf('noopener') === -1) {
+      relList.push('noopener');
+    }
+    if (relList.indexOf('noreferrer') === -1) {
+      relList.push('noreferrer');
+    }
+    domProps.rel = relList.join(' ');
```

Comment:
@necolas, how's this? We have only one array allocation now. Want me to keep it in pure string land?